cpp/lib/include/AppArmorUtil.h

```diff
@@ -40,11 +40,13 @@ std::vector<std::pair<std::string, ProfileMode>> GetProfiles();
 /** \brief  Return the mode for the given profile. */
 ProfileMode GetProfileMode(const std::string &profile_id);
 
-/** \brief  Install the profile from the given path. Filename will later be used as profile_id. */
-void InstallProfile(const std::string &profile_path);
+/** \brief  Install the profile from the given path as a local profile.
+ *          The filename of the profile_path will later be used as the profile_id.
```

Comment:
Shouldn't that read "as the profile ID"?

Reply:
Also after looking at the implementation, I would shy away from using "path" as that implies a path.  How about profile_config_filename or something similar?

Reply:
Everytime i read a variable called "filename", i assume this means only the name of the file without the full path, which is mandatory in this case.